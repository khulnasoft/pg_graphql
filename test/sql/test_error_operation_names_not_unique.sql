select graphql.resolve(
    query:='query ABC { anon }
            query ABC { other }'
)
