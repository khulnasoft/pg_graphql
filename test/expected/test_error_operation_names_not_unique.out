select graphql.resolve(
    query:='query ABC { anon }
            query ABC { other }'
)
                           resolve                           
-------------------------------------------------------------
 {"errors": [{"message": "Operation names must be unique"}]}
(1 row)

